src/nl/documentatie/14.shopify.md

```diff
@@ -52,60 +43,92 @@ Om de MyParcel-plugin te installeren, volg je de volgende stappen:
 
 De app wordt bij wijzigingen automatisch bijgewerkt, je hebt hier zelf geen onderhoud aan.
 
+### Eenmalige handeling
+
+<MPImg src="/documentation/shopify/nl/shopify-api-key.jpg" alt="Shopify MyParcel API setting" />
+
+Voor het in gebruik nemen van de MyParcel app, moet er eerst een API key worden ingevuld. De API key is gratis te verkrijgen via <a href="https://backoffice.myparcel.nl/registration" target="_blank"> Shopinstellingen > API instellingen in de MyParcel Backoffice</a>.
+Na het invullen van de API key wordt de MyParcel app gekoppeld aan jouw Shopify shop.
```

Comment:
```suggestion
Na het invullen van de API key wordt de MyParcel app gekoppeld aan jouw Shopify shop.
```
De MyParcel app wordt gekoppeld aan jouw Shopify shop wanneer de API key is ingevuld.